core/src/main/scala/fs2/PullDerived.scala

```diff
@@ -6,9 +6,9 @@ trait PullDerived { self: fs2.Pull.type =>
    * Acquire a resource within a `Pull`. The cleanup action will be run at the end
    * of the `.run` scope which executes the returned `Pull`.
```

Comment:
Might want to add a comment about the meaning of the `Pull` that is returned as well.